src/Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs

```diff
@@ -2800,15 +2800,16 @@ private void GenerateHelperFunctions(TextWriter writer)
 
```

Comment:
@vexx32, your last commit had 3 failures in `PowerShell-CI-windows`
Remoting loopback tests.Can execute command in a disconnected session
```powershell
Objects cannot be added to a closed buffer. Make sure the buffer is open for Add and Insert operations to succeed.
at <ScriptBlock>, D:\a\1\s\test\tools\Modules\HelpersRemoting\HelpersRemoting.psm1: line 226
226:     Invoke-Command @parameters

```
Tests for (error, warning, etc) action preference.Setting ErrorActionPreference to stop prevents user from getting the error exception.$err.Count
```powershell
Expected 1, but got 2.
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Scripting\ActionPreference.Tests.ps1: line 79
79:         It '$err.Count' { $err.Count | Should -Be 1 }
```
Function Pipeline Behaviour.Output Behaviour on Error States.still completes Cleanup {} execution when StopProcessing() is triggered mid-Cleanup {}
```powershell
Expected exactly 'CLEANUP', but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Scripting\PipelineBehaviour.Tests.ps1: line 119
119:             $powershell.Streams.Information[0].MessageData | Should -BeExactly "CLEANUP"
```